.github/workflows/scripts/start-rancher.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+flip() {
+    echo $1
+    exit 1
+}
+
+kubectl get ns | grep -s cattle && flip 'rancher already installed?'
+
+set -exu
+
+set +e
+helm repo add cert-manager https://charts.jetstack.io
+helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+helm repo add jetstack https://charts.jetstack.io
+set -e
+
+helm repo update
+
+helm upgrade --install cert-manager --namespace cert-manager cert-manager/cert-manager --set installCRDs=true --set "extraArgs[0]=--enable-certificate-owner-ref=true" --create-namespace --wait --timeout=10m
```

Comment:
That's been part of the command-line I've used to install `cert-manager` since the first time I installed Rancher when testing Rancher Desktop.

Reply:
I don't see that argument as part of our [docs](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster#4-install-cert-manager). Personally I don't think I've ever used it. I think it would be good to avoid including it if we don't need it